NEWS.md

```diff
@@ -151,6 +151,8 @@ New language features
 
   * `@__LINE__` special macro now available to reflect invocation source line number ([#12727]).
 
+  * `PROGRAM_FILE` global is now available for determining the name of the running script ([#14114]).
```

Comment:
Hey, I just noticed that this was accidentally added to the Julia 0.4 NEWS, not the Julia 0.5 NEWS...